.ddev/config.yaml

```diff
@@ -0,0 +1,292 @@
+name: matomo
+type: php
+docroot: ""
+php_version: "7.2"
+webserver_type: apache-fpm
+xdebug_enabled: false
+additional_hostnames: []
+additional_fqdns: []
+database:
+    type: mysql
+    version: "5.7"
+hooks:
+    post-start:
+        - exec: MYSQL_PWD=root mysql -uroot -e "GRANT ALL PRIVILEGES ON *.* to 'db'@'%'"
+          service: db
+        - exec-host: ddev matomo:init
+webimage_extra_packages: [build-essential, chromium, fonts-dejavu, fonts-liberation, gconf-service, imagemagick, imagemagick-doc, libappindicator1, libasound2, libatk1.0-0, libcairo2, libdrm2, libgbm1, libgconf-2-4, libgdk-pixbuf2.0-0, libgtk-3-0, libnspr4, libnss3, libpango-1.0-0, libpangocairo-1.0-0, libx11-xcb1, libxcomposite1, libxcursor1, libxdamage1, libxfixes3, libxi6, libxrandr2, libxrender1, libxshmfence1, libxss1, libxtst6, ripgrep, ttf-mscorefonts-installer, xdg-utils]
+use_dns_when_possible: true
+composer_version: "2"
+web_environment:
+    - MATOMO_DATABASE_HOST=db
+    - MATOMO_DATABASE_USERNAME=db
+    - MATOMO_DATABASE_PASSWORD=db
+    - MATOMO_DATABASE_DBNAME=db
+nodejs_version: "16"
```

Comment:
This should set the system NodeJS version to 16.x, allowing to run `vue:build` right away after startup.

Reply:
works as expected